documentation/modules/post/networking/gather/enum_juniper.md

```diff
@@ -59,7 +59,7 @@
 [+] radius server 1.1.1.1 password hash: $9$Y-4GikqfF39JGCu1Ileq.PQ6AB1hrlMBIyKvWdV
 [+] PPTP username 'pap_username' hash $9$he4revM87-dsevm5TQCAp0BErvLxd4JDNdkPfT/9BIR via PAP
 [*] Post module execution completed
-msf5 post(juniper/gather/enum_juniper) > creds
```

Comment:
The documentation here doesn't actually show how we configured this module prior to running it. We should probably fix this up.

Reply:
addressed in commit